boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: nrf54h20dk/nrf54h20/cpuapp
+name: nRF54H20-DK-nRF54H20-Application
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+  - zephyr
+ram: 32
+flash: 368
```

Comment:
these numbers can't really be accurate can they?

Reply:
ram: 256k

Reply:
Updated.